src/test/resources/log4j2-test.xml

```diff
@@ -7,12 +7,8 @@
     </Appenders>
 
     <Loggers>
-
-        <!-- Apache Commons tend to make a lot of noise which can clutter the log-->
-        <AsyncLogger name="org.apache" level="WARN"/>
-
-        <!-- Reduce startup noise -->
-        <AsyncLogger name="org.springframework.beans.factory" level="WARN"/>
+        <!-- HTTP Wire logging -->
+        <AsyncLogger name="org.mule.service.http.impl.service.HttpMessageLogger" level="INFO" />
```

Comment:
Was this really intentional?

Reply:
Yes, the spring/apache ones seem to be just copy-pasted from the runtime's default config, and having the wire-logging in INFO here makes it faster to use it when troubleshooting the tests (by changing it to DEBUG)